src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  /* display: flex; */
+  background-image: url("./images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  /* background-size: contain; */
+  /* background-size: 13px 13px; */
+  background-size: auto;
+  color: #000;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
```

Comment:
and this?

Reply:
border-radius: 0.5px;
from Figma - removed

  height: 16px;
  width: 16px;
stars don't show without those properties - should I remove them?

Reply:
+